aztec/src/main/java/org/wordpress/aztec/Html.java

```diff
@@ -751,28 +751,6 @@ private static void end(SpannableStringBuilder text, TextFormat textFormat) {
         }
     }
 
-//<<<<<<< HEAD
```

Comment:
Good idea.  We can remove [this unused code](https://github.com/wordpress-mobile/WordPress-Aztec-Android/pull/246/files#diff-7177c376b871e9e1aa2e4983072f0705L164) too.